apl/scalar-functions/string-functions/translate.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: translate
+description: 'This page explains how to use the translate function in APL.'
+---
+
+Use the `translate` function in APL (Axiom Processing Language) to substitute characters in a string, one by one, based on their position in two input lists. For every character in the input string that matches a character in the first list, `translate` replaces it with the character at the same position in the second list.
```

Comment:
this function also has a useful quirk
> // quirks: if replace is shorter than the search, then the last character is repeated to match the length of the search

so 
```kql 
translate('abc', 'x', "abcdef") => "xxxdef"
translate('abc', 'xy', "abcdef') => "xyydef"
```

it also accepts empty strings for the second parameter so 
```kql
translate('abc', '', 'abcdefabc') => "abc"
```